lib/samwise/client.rb

```diff
@@ -32,6 +40,18 @@ def excluded?(duns: nil)
 
     def small_business?(duns: nil)
       response = lookup_duns(duns: duns)
+      parse_response_for_small_business(response)
+    end
+
+    private
+
+    def parse_response_for_sam_status(response)
+      response.status == 200
+    end
+
+    def parse_response_for_small_business(response)
+      return false if response.code == 404
```

Comment:
Do you want to add a `rescue` or does it handle things like JSON parsing exceptions somewhere else? (Is that the caller's responsibility?)